garrysmod/gamemodes/terrortown/gamemode/cl_lang.lua

```diff
@@ -151,7 +151,7 @@ function LANG.SetActiveLanguage(lang_name)
 end
 
 function LANG.Init()
-   local lang_name = GetConVarString("ttt_language")
+   local lang_name = GetConVar:GetString("ttt_language")

```

Comment:
FFS, read the documentation. Cache it OUTSIDE the function, then run GetString on the local